INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 249-222-279
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [23/Oct/2019 17:16:04] "GET / HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [23/Oct/2019 17:16:04] "GET /favicon.ico HTTP/1.1" 404 -
DEBUG app_user.py:9 eeeeeeeeee
INFO _internal.py:122 127.0.0.1 - - [23/Oct/2019 17:16:44] "GET /api/user/index HTTP/1.1" 200 -
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [24/Oct/2019 10:58:50] "GET / HTTP/1.1" 404 -
DEBUG app_user.py:9 eeeeeeeeee
INFO _internal.py:122 127.0.0.1 - - [24/Oct/2019 10:58:57] "GET /api/user/index HTTP/1.1" 200 -
DEBUG app_user.py:9 eeeeeeeeee
INFO _internal.py:122 127.0.0.1 - - [24/Oct/2019 10:59:00] "GET /api/user/index HTTP/1.1" 200 -
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
DEBUG app_user.py:9 eeeeeeeeee
INFO _internal.py:122 127.0.0.1 - - [24/Oct/2019 10:59:36] "GET /api/user/index HTTP/1.1" 200 -
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
DEBUG app_user.py:9 eeeeeeeeee
INFO _internal.py:122 127.0.0.1 - - [24/Oct/2019 11:00:57] "GET /api/user/index HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [24/Oct/2019 11:00:58] "GET /favicon.ico HTTP/1.1" 404 -
INFO _internal.py:122  * Detected change in 'D:\\blog\\myBlog\\back_end\\app\\__init__.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
DEBUG app_user.py:9 eeeeeeeeee
INFO _internal.py:122 127.0.0.1 - - [24/Oct/2019 11:01:31] "GET /api/user/index HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'D:\\blog\\myBlog\\back_end\\app\\__init__.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
DEBUG app_user.py:9 eeeeeeeeee
INFO _internal.py:122 127.0.0.1 - - [24/Oct/2019 11:01:46] "GET /api/user/index HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'D:\\blog\\myBlog\\back_end\\app\\__init__.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
DEBUG app_user.py:9 eeeeeeeeee
INFO _internal.py:122 127.0.0.1 - - [24/Oct/2019 11:03:43] "GET /api/user/index HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'D:\\blog\\myBlog\\back_end\\app\\__init__.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'D:\\blog\\myBlog\\back_end\\utils\\log.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
DEBUG app_user.py:9 eeeeeeeeee
INFO _internal.py:122 127.0.0.1 - - [24/Oct/2019 11:05:48] "GET /api/user/index HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'D:\\blog\\myBlog\\back_end\\config.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
DEBUG app_user.py:9 eeeeeeeeee
INFO _internal.py:122 127.0.0.1 - - [24/Oct/2019 11:06:31] "GET /api/user/index HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'D:\\blog\\myBlog\\back_end\\config.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'D:\\blog\\myBlog\\back_end\\utils\\log.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
ERROR __init__.py:97 Error: Directory migrations already exists and is not empty
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [28/Oct/2019 11:26:25] "POST /api/user/image_code HTTP/1.1" 403 -
INFO _internal.py:122  * Detected change in 'D:\\blog\\myBlog\\back_end\\app\\blog_api\\app_user.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [28/Oct/2019 11:26:52] "POST /api/user/image_code HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'D:\\blog\\myBlog\\back_end\\app\\blog_api\\app_user.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [28/Oct/2019 11:28:14] "GET /api/user/image_code HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [28/Oct/2019 11:28:14] "GET /api/user/image_code?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Oct/2019 11:28:14] "GET /api/user/image_code?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Oct/2019 11:28:14] "GET /api/user/image_code?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Oct/2019 11:28:14] "GET /api/user/image_code?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Oct/2019 11:28:14] "GET /api/user/image_code?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Oct/2019 11:28:14] "GET /api/user/image_code?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'D:\\blog\\myBlog\\back_end\\app\\blog_api\\app_user.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [28/Oct/2019 11:28:37] "GET /api/user/image_code HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Oct/2019 11:28:38] "GET /favicon.ico HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [28/Oct/2019 11:28:56] "GET /api/user/image_code HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Oct/2019 11:29:08] "GET /api/user/image_code HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'D:\\blog\\myBlog\\back_end\\utils\\captcha\\captcha.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 302-102-607
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'D:\\blog\\myBlog\\back_end\\app\\__init__.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 249-222-279
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'D:\\blog\\myBlog\\back_end\\app\\__init__.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 249-222-279
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'D:\\blog\\myBlog\\back_end\\app\\__init__.py', reloading
INFO _internal.py:122  * Restarting with stat
